### ISSUE TITLE
bigImageUrl Extra Possibly Not Working

### ISSUE THREAD

**Issue by opsoyo** [gotify/android#204]
For 8f730bacceed27514f74eafb8a5ddf28f650457e, I figured the usage would be:
```
{
  "extras": {
    "client::notification": {
      "bigImageUrl": "https://i.imgur.com/gRo7tfw.jpg",
      "click": { "url": "https://gotify.net" }
    }
  },
  "message": "Big images are fun",
  "priority": 5,
  "title": "Big Image Test"
}
```
This is what happens instead:

![](https://i.imgur.com/IvIitiT.png)

Verified both the server and Android app are latest releases.

**Comment by jmattheis** [comment]
It does work for me. Which version of the android app do you have, do you have the playstore or fdroid version?

![image](https://user-images.githubusercontent.com/14895212/148684257-a1042cd5-1d17-4f85-9f7c-f5e563347f94.png)

**Comment by jmattheis** [comment]
Ahh, nvm, this isn't released yet xD.

**Comment by jmattheis** [comment]
I've released [2.3.0](https://github.com/gotify/android/releases/tag/v2.3.0). The update in google play / f-droid takes some time, but you could use the APK in the release.

**Comment by opsoyo** [comment]
Fantastic! Thank you. I can confirm that it works with that updated APK.
